src/main/java/excel/export/service/excel/ExportToExcelFileService.java

```diff
@@ -0,0 +1,75 @@
+package excel.export.service.excel;
+
+import excel.export.model.ParkingLot;
+import excel.export.service.ParkingLotService;
+import excel.export.transformer.ParkingLotToXlsRowTransformer;
+import lombok.extern.slf4j.Slf4j;
+import org.apache.poi.hssf.usermodel.HSSFWorkbook;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Service;
+
+import java.util.Collections;
+import java.util.List;
+
+import static java.util.stream.Collectors.toList;
+
+@Slf4j
+@Service
+public class ExportToExcelFileService {
+    private ParkingLotService parkingLotService;
+    private ParkingLotToXlsRowTransformer parkingLotTransformer;
+
+    @Autowired
+    public ExportToExcelFileService(ParkingLotService parkingLotService, ParkingLotToXlsRowTransformer parkingLotTransformer) {
+        this.parkingLotService = parkingLotService;
+        this.parkingLotTransformer = parkingLotTransformer;
+    }
+
+    public HSSFWorkbook getExcelFile() {
+        List<ParkingLot> allParkingLots = parkingLotService.getAllParkingLots();
+        List<Object[]> excelRows = allParkingLots.stream()
+                .map(parkingLot -> parkingLotTransformer.transform(parkingLot))
+                .collect(toList());
+
+        return new XlsBuilder()
```

Comment:
This part of the code may be extracted into the function. This will eliminate code duplication.

Reply:
Ok, then it'll be

Reply:
Ok.